bootstrap_salt/fab_tasks.py

```diff
@@ -52,6 +55,47 @@ def find_master():
     return master
 
 
+def set_master_dns():
+    master_ip = find_master()
+    project_config = config.ProjectConfig(env.config,
+                                          env.environment,
+                                          env.stack_passwords)
+    cfg = project_config.config
+    try:
+        zone_name = cfg['master_zone']
+    except KeyError:
+        logging.warn("master_zone not found in config file, "\
+                     "no DNS entry will be created, you will "\
+                     "need AWS access to deploy etc.")
+        return False
+    r53 = get_connection(R53)
+    zone_id = r53.get_hosted_zone_id(zone_name)
+    dns_name = 'master.{0}.{1}.{2}'.format(env.environment,
+                                           env.application,
+                                           zone_name)
+    r53.update_dns_record(zone_id, dns_name, 'A', master_ip)
+    return True
+
+
+def get_master_name():
```

Comment:
Hmm, do you think it's more readable in one function. It would also mean that I don't need to change template deploy (other than to bump the required version of bootstrap-salt). So I'd say on balance probably a good idea.